vendor/assets/stylesheets/spree/frontend/global.css.scss

```diff
@@ -625,6 +625,22 @@ a.btn {
     background: linear-gradient(167deg, #a71e22 16%, #a71e22 82%);
 }
 
+.prev-page-icon,

```

Comment:
Why we need to add these extra classes. Any specific reason?

Reply:
Because we need to adjust previous page design as per next page design if we dont add these classes it disturbs design and it will not be according to template

Reply:
I dnt agree with this point.  Please discuss it with Umer. As we have all the design. We dnt need to add any extra class. If you need to add , then discuss it with me or Umer before implementation

Reply:
I dnt agree with this point. Please discuss it with Umer. As we have all the design. We dnt need to add any extra class. If you need to add , then discuss it with me or Umer before implementation